Symfony/CS/Tests/Tokenizer/TokenTest.php

```diff
@@ -174,6 +174,13 @@ public function testIsKeywords()
         $this->assertFalse($this->getBraceToken()->isKeyword());
     }
 
+    public function testIsMagicConstant()
+    {
+        $this->assertFalse($this->getBraceToken()->isMagicConstant());
+        $token = new Token(array(T_CLASS_C, '__CLASS__'));
+        $this->assertTrue($token->isMagicConstant());
```

Comment:
added in https://github.com/SpacePossum/PHP-CS-Fixer/commit/96a7ac99d74f8a9504c1648101088937ba19972c with some more test cases.